CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ Otherwise later tooling on CI may yell at you about formatting/linting violation
 
 Some targets are generated from sources.
 Currently this is just the `bzl_library` targets.
-Run `bazel run //:gazelle` to keep them up-to-date.
+Run `aspect configure` to keep them up-to-date.
```

Comment:
Is workflows CI already verifying `aspect configure` doesn't return a diff? I believe the `gazelle` task also adds a test verifying that...

Reply:
yes I think that is rolled out in the latest workflows release so we can be confident the BUILD files already stay updated.

Maybe we do need to add a `configure` task to the OSS repos as I roll this out though.

Reply:
Adding `configure` is what I thought we might need in these PRs

Reply:
This repo isn't on aspect workflows yet, so there's no CI coverage for `bzl_library` being up-to-date.
Will add as a follow-up.